content/blog/eng/lambda-days-2020.md

```diff
@@ -0,0 +1,91 @@
+---
+authors: ["andreac"]
+comments: true
+date: "2020-04-09"
+draft: false
+share: true
+categories: [English, Conference, Lambda-days, Krakow, Functional programming]
+title: "Lambda-days-2020"
+ita: "Lambda-days-2020"
+
+languageCode: "en-US"
+type: "post"
+---
+
+# Lambda Days 2021
+
+This year, Facile attended for the first time to [Lambda Days](https://www.lambdadays.org/lambdadays2020) Conference in the beautiful polish city of Krakow.
+It has been a two day full immersion, where the organizers have been able to gather speakers from all over the world
+showcasing the latest trends and academic research in the functional world, spanning from beginner approaches to functional languages 
+to super specific scientific applications.
+
+This conference totally worth attending, albeit very intense (sometimens mind blowing, actually!), the talks were high quality, and perfect organization.
+
+Here is a short list of talks that took our attention:
+
+- **Keynote: How to specify it! A guide to writing properties of pure functions**
+
+[Video](https://www.youtube.com/watch?v=G0NUOst-53U&list=PLvL2NEhYV4ZsV9Bw0wp1P46SOdtk4pFW6&index=3&t=0s)
+
+**John Hughes** one of Haskell creators presented his QuickCheck library
+In a funny manner, he showed how use his library to create automated test-cases and gave some tips on how to write tests that actually
```

Comment:
In a funny way, he showed how to use his library to create automated test-cases. He also gave us some tips on how to write tests that really verify assumptions and don't give a false green check back.